packages/web3-providers-http/src/index.ts

```diff
@@ -0,0 +1,58 @@
+import axios, { AxiosInstance } from 'axios';
+import Web3ProviderBase from 'web3-providers-base';
+import { ProviderOptions } from 'web3-providers-base/types';
```

Comment:
We could export the types in the root, and allow you to pull them all in the same import statement. I don't have a real preference over this though so up to you!

Reply:
Mmm I'm not sure on this one

Reply:
Actually, I tried this out, and I think I like separating type imports because it's explicit that they're types - going to revert my changes and keep it how it is, unless someone believes otherwise is better